alerts/host_scanner_alerts.py

```diff
@@ -1,43 +0,0 @@
-#!/usr/bin/env python
```

Comment:
This is deprecated as this solution was modified and lives on in ssh_key.py, thus removing